packages/dds/legacy-dds/src/array/interfaces.ts

```diff
@@ -63,11 +64,13 @@ export interface ISharedArrayEvents extends ISharedObjectEvents {
  * @alpha
  */
 export interface ISharedArray<T extends SerializableTypeForSharedArray>
-	extends ISharedObject<ISharedArrayEvents> {
+	extends SharedObject<ISharedArrayEvents> {
```

Comment:
i don't think this is correct. i don't think this interface should extend that class. why is this necessary?

Reply:
<img width="820" alt="image" src="https://github.com/user-attachments/assets/c7d1ae2c-87d4-43a3-8a73-2ec09a8282c6" />